config/datocms/migrations/1735398092_emailAndPhoneLinks.ts

```diff
@@ -0,0 +1,235 @@
+import type { Client } from '@datocms/cli/lib/cma-client-node';
+import { actionStyleField } from '../lib/fields';
+
+export default async function (client: Client) {
+  console.log('Manage upload filters');
+
+  console.log('Install plugin "Dropdown Conditional Fields"');
+  await client.plugins.create({
+    id: 'Srdwo4YOREmRtvMAV2otlQ',
+    package_name: 'datocms-plugin-dropdown-conditional-fields',
+  });
+  await client.plugins.update('Srdwo4YOREmRtvMAV2otlQ', {
+    parameters: { developmentMode: false },
+  });
+
+  console.log('Create new models/block models');
+
+  console.log('Create block model "\uD83D\uDCDE Phone Link" (`phone_link`)');
+  await client.itemTypes.create(
+    {
+      id: 'C5fWG5CYRJ69oqaP6CjYdA',
+      name: '\uD83D\uDCDE Phone Link',
+      api_key: 'phone_link',
+      modular_block: true,
+      inverse_relationships_enabled: false,
+    },
+    {
+      skip_menu_item_creation: true,
+      schema_menu_item_id: 'YNbDA4R-Srqf0u4-Rya3Ug',
+    }
+  );
+
+  console.log('Create block model "\uD83D\uDCE7 Email Link" (`email_link`)');
+  await client.itemTypes.create(
+    {
+      id: 'b90_c2zeS6auRELEzZHNcA',
+      name: '\uD83D\uDCE7 Email Link',
+      api_key: 'email_link',
+      modular_block: true,
+      inverse_relationships_enabled: false,
+    },
+    {
+      skip_menu_item_creation: true,
+      schema_menu_item_id: 'Vgf9ZeHvTqeh-tMAks2jFw',
+    }
+  );
+
+  console.log('Creating new fields/fieldsets');
+
+  console.log(
+    'Create Single-line string field "Title" (`title`) in block model "\uD83D\uDCDE Phone Link" (`phone_link`)'
+  );
+  await client.fields.create('C5fWG5CYRJ69oqaP6CjYdA', {
+    id: 'fp9Cugu8QlKqawis3QTnPA',
+    label: 'Title',
+    field_type: 'string',
+    api_key: 'title',
+    validators: { required: {} },
+    appearance: {
+      addons: [],
+      editor: 'single_line',
+      parameters: { heading: false, placeholder: null },
+    },
+    default_value: '',
+  });
+
+  console.log(
+    'Create Single-line string field "Phone number" (`phone_number`) in block model "\uD83D\uDCDE Phone Link" (`phone_link`)'
+  );
+  await client.fields.create('C5fWG5CYRJ69oqaP6CjYdA', {
+    id: 'bfnOOp5cSM-x8S5RKEpKsw',
+    label: 'Phone number',
+    field_type: 'string',
+    api_key: 'phone_number',
+    hint: 'Best to use international notation: +31 20 2610954',
+    validators: { required: {} },
```

Comment:
Could add more validation here, but validating phone numbers can be tricky. Editors can also just try if the number the entered, works in the preview.

Even more nice to have would be a Phone Input Field as a DatoCMS plugin. We could create that based on [`react-phone-number-input`](https://www.npmjs.com/package/react-phone-number-input).

Reply:
I would even go as far as saying this feature can be implemented just by adding a plugin, right? They're all links..

Reply:
@MarleenEliza mentioned that my assumption was incorrect, `mailto:` etc are note valid URLs according to Dato.